frontend/src/themes/components/button.ts

```diff
@@ -136,6 +136,40 @@ const Button = {
       lineHeight: "21px",
       letterSpacing: "0",
     },
+    delete: {
+      bg: "red.200",
+      color: "#FFFFFF",
+      _active: {
+        bg: "red.100",
+        border: "none",
+      },
+      _hover: {
+        bg: "red.100",
+        border: "1.5px solid #FFFFFF",
+        _disabled: {
+          bg: "red.100",
+          border: "none",
+        },
+      },
+      WhiteIcon,
+    },
+    deleteCancel: {
+      bg: "red.50",
+      color: "red.200",
+      _active: {
```

Comment:
maybe we can factor out the active and hover styling into a constant

Reply:
good idea! im gonna do that in my next pr though cuz im blocking like 3 people with this pr 😅